vtds_application_openchami/private/application.py

```diff
@@ -268,6 +267,17 @@ def consolidate(self):
             )
         self.config['discovery_networks'] = filtered_discovery_networks
 
+        # Clean out RIE servers that are marked for deletion and clean
+        # out the 'delete' key value pair from all RIE servers
+        self.config['rie_services'] = {
+            name: {
+                key: value
+                for key, value in service.items() if key != 'delete'
+            }
+            for name, service in self.config.get('rie_services', {}).items()
+            if not service.get('delete', False)
```

Comment:
Just a knit pick - it would be easier to read if dangling `if` was indented (if not in the same line as above).